orc8r/cloud/docker/fluentd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fluent/fluentd:v1.7-1
+FROM fluent/fluentd:v1.14.6-debian-1.0
```

Comment:
Debian is necessary here due to the `fluent-plugin-systemd` dependency?

Reply:
Frankly speaking, it is the only one that worked for me. E.g. it returned an error stating it could not find 1.14-1. (the fluentd_daemon Dockerfile specified Debian before the update as well, the others didn't)

Reply:
Weird. The [v1.14-1 image is available at docker-hub](https://hub.docker.com/layers/fluentd/library/fluentd/v1.14-1/images/sha256-9624d5acdaf03449120293a1f5332154dc9c2796fddf4b1f998aeee5aaca7027?context=explore) and I have no issue pulling it.
![grafik](https://user-images.githubusercontent.com/13189449/177716859-b459fb8f-c2fb-4396-b4bc-f2835540d609.png)

Reply:
I tried again, and the problem only occurs when building the fluentd-daemon image. If I change the version to 1.14-1 in `.../fluentd_daemon/Dockerfile`, the build process cancels with this error:
![Screenshot from 2022-07-08 09-41-45](https://user-images.githubusercontent.com/104822369/177942979-9d213155-d256-463e-aeec-db75c119b84e.png)
If I use the version 1.14.6-debian-1.0 it works fine. The other images are not affected by this, i.e. using the other version in the respective Dockerfiles still works. Maybe this is the reason the debian package was used for the fluentd-daemon image before the update as well?
I changed it for the others because I think it might be better/more consistent to use the same package everywhere.

Reply:
Thanks for the additional comment. So maybe I misunderstood you previous comment?

__To clarify:__
There is no problem with image availability for `fluentd:v1.14-1`.
The issue is quite clearly stated in your console output, that the `apt-get` command is not available in the `fluentd:v1.14-1` image. But this is expected, as the `fluentd:v1.14-1` image is an [Alpine Linux Image](https://github.com/fluent/fluentd-docker-image/blob/master/v1.14/alpine/Dockerfile#L4), which does not provide `apt-get`.

I.e. the custom installation steps are not designed to be run on Alpine. If this is desired, one would need to port that to Alpine, which would be a separate effort.

__FYI:__
To check for an image OS you could also download it and login to access the OS version.
```bash
HOST $ docker pull fluentd:v1.14-1 # download image
HOST $ docker run --rm -i -t fluentd:v1.14-1 sh # run image and login (be aware that alpine has no bash)
ALPINE $ cat /etc/os-release 
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.13.10
PRETTY_NAME="Alpine Linux v3.13"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
```

Reply:
Ok, thanks a lot for the clarification and the information!